jaxrs/security/resteasy-crypto/pom.xml

```diff
@@ -52,15 +52,16 @@
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk16</artifactId>
+            <artifactId>bcprov-jdk15on</artifactId>
         </dependency>
         <dependency>
            <groupId>org.bouncycastle</groupId>
-           <artifactId>bcmail-jdk16</artifactId>
+           <artifactId>bcmail-jdk15on</artifactId>
         </dependency>
         <dependency>
-            <groupId>javax.mail</groupId>
-            <artifactId>mail</artifactId>
+            <groupId>com.sun.mail</groupId>
+            <artifactId>javax.mail</artifactId>
+            <version>1.5.3</version>

```

Comment:
no need for version here it is defined in parent pom, otherwise use ${dep.com.sun.mail.version}